app/screens/Todo/index.jsx

```diff
@@ -1,21 +1,246 @@
+                if (!onEndReachedCalledDuringMomentum) {
+                  onEndReachedHandler();
+                  setOnEndReachedCalledDuringMomentum(true);
+                }
+              }}
+              onEndReachedThreshold={0.1}
+            />
+          </View>
+          /* tab === 'DONE' ? (
+            <View style={styles.body}>
+              <FlatList
+                data={doneTodo}
+                renderItem={({item}) => <ListItem todo={item} />}
+                refreshControl={
+                  <RefreshControl
+                    refreshing={isLoading}
+                    onRefresh={onRefresh}
+                  />
+                }
+                onEndReached={onEndReachedHandler}
+                onEndReachedThreshold={0}
+                ListFooterComponent={renderLoader}
+              />
+            </View>
+          ) : (
+            <View style={styles.body}>
+              <FlatList
+                data={todos}
+                renderItem={({item}) => <ListItem todo={item} />}
+                refreshControl={
+                  <RefreshControl
+                    refreshing={isLoading}
+                    onRefresh={onRefresh}
+                  />
+                }
+                onEndReached={onEndReachedHandler}
+                onEndReachedThreshold={0}
+                ListFooterComponent={renderLoader}
+              />
+            </View> */
+        }
+
+        <TouchableOpacity style={styles.button} onPress={onAddPressHandler}>
+          <Icon name="plus-a" size={20} />
+        </TouchableOpacity>
+      </SafeAreaView>
+    </LinearGradient>
   );
 };
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
+  },
+  bigText: {
+    fontWeight: 'bold',
+    fontSize: 25,
+    color: 'black',
+  },
+  smallText: {
+    fontWeight: 'bold',
+    fontSize: 15,
+    color: 'black',
+  },
+  header: {
+    padding: 15,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    backgroundColor: '#9395D3',
+    borderBottomLeftRadius: 30,
+    borderBottomRightRadius: 30,
+  },
+  focusedText: {
+    fontWeight: 'bold',
+    textDecorationLine: 'underline',
+    fontSize: 30,
+    color: '#fff',
+  },
+  unFocusedText: {
+    fontSize: 30,
+    color: '#fff',
+  },
+  body: {
+    flex: 1,
+    marginTop: 15,
+    justifyContent: 'center',
     alignItems: 'center',
+  },
+  button: {
+    width: 60,
+    height: 60,
+    position: 'absolute',
+    bottom: 20,
+    right: 10,
+    backgroundColor: '#B3B7EE',
+    borderRadius: 40,
+    alignItems: 'center',
+    justifyContent: 'center',
+    elevation: 5,
+  },
+  actionIcon: {
+    width: 30,
+    height: 30,
     justifyContent: 'center',
+    borderRadius: 3,
+    alignSelf: 'center',
+    marginLeft: 5,
+    marginRight: 5,
+  },
+  listItem: {
+    padding: 10,
+    backgroundColor: 'white',
+    flexDirection: 'row',
+    borderRadius: 7,
+    marginVertical: 9,
+    width: 350,
```

Comment:
It should be removed. cuz we already used `FlatList` and the width is full width.
```tsx
// ....
  listItem: {
    padding: 10,
    backgroundColor: 'white',
    flexDirection: 'row',
    borderRadius: 7,
    marginVertical: 9,
    // width: 350,
    marginHorizontal: 20,
    elevation: 3,
  },
// ....
```